cate/core/ds.py

```diff
@@ -338,6 +338,18 @@ def invalidate(self):
         """
         pass
 
+    def get_updates(self, reset=False) -> Dict:
+        """
+        Ask the datastore to retrieve the differences found between a previous
+        dataStore status and the current one,
+        The implementation should return a dictionary with the new ['new'] and removed ['del'] dataset.
+        it also return the reference time to the datastore status taken as previous, it could
+        be useful to know to which time the found differences are compared to.
```

Comment:
Avoid "could" in operation specification. Is the framework using the reference time or not? Be clear.